src/PoCompiler.php

```diff
@@ -273,7 +265,8 @@ protected function cleanExport($string)
      */
     private function wrapString($value)
     {
-        $wrapped = \wordwrap($value, $this->wrappingColumn, " \n");
-        return \explode("\n", $wrapped);
+        $value = str_replace("\n", chr(13) . chr(28), $value);
```

Comment:
I feel like `chr(13)`, `chr(28)` and similar are hard to read. At least to know what characters they are representing.  
I would create variables for them with auto-explanative names.  
Example:
```
$carriageReturn = chr(13);
$somethingExplanatori = chr(28);
```

Reply:
You're totally right. I've fixed that. See latest commit.